config/rbac/leader_election_role.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: leader-election-role
 rules:
 - apiGroups:
-  - ""
+  - "coordination.k8s.io
```

Comment:
```suggestion
  - "coordination.k8s.io"
```

Reply:
are you sure `coordination.k8s.io` about this as when I tested druid before release I didn’t face any error.

Reply:
Fixed